config/default/webform.webform.wizard_feedback.yml

```diff
@@ -229,7 +232,7 @@ handlers:
     handler_id: foia_submission_queue
     label: 'FOIA Submission Queue'
     notes: ''
-    status: true
+    status: false
```

Comment:
What does this change do?

Reply:
It seems to enable/disable the handler that "Queues a webform submission to be sent later".  When I edited the form last time, this setting changed automatically, but perhaps it is best to leave it enabled until we're told otherwise since it seems to be a default setting.